.github/workflows/tests.yml

```diff
@@ -6,6 +6,11 @@ on:
     branches:
       - main
       - release-*
+    paths:
```

Comment:
Maybe using `paths-ignore: docs/**` is a bit more robust?

Reply:
Eh, it's the same I think, as we would still need to list the README, changelog, etc...

Reply:
I was working on the assumption that it is better to run the tests once more that to do not run them if the patterns here are not updated.

Reply:
That's true, and given I don't think we are gonna be able to get Github to display a *skiped* icon on the check, I think that may be better. Thanks.